front/app/modules/commercial/granular_permissions/admin/containers/Granular/ActionsForm.tsx

```diff
@@ -1,36 +1,56 @@
 import React, { memo } from 'react';
 import styled from 'styled-components';
 import { isEmpty } from 'lodash-es';
-import { isNilOrError } from 'utils/helperUtils';
 
 // services
 import {
-  IPermissionData,
   IGlobalPermissionAction,
-  IPCPermissionAction,
+  IPermissionData,
 } from 'services/actionPermissions';
-import { getInputTerm } from 'services/participationContexts';
 
 // components
 import ActionForm from './ActionForm';
+import UserFieldSelection from '../../components/UserFieldSelection/UserFieldSelection';
+import { Box, colors, Title } from '@citizenlab/cl2-component-library';
 
 // i18n
-import { FormattedMessage, MessageDescriptor } from 'utils/cl-intl';
+import { FormattedMessage } from 'utils/cl-intl';
 import messages from './messages';
-import { getInputTermMessage } from 'utils/i18n';
 
 // hooks
 import useProject from 'hooks/useProject';
 import usePhases from 'hooks/usePhases';
+import useFeatureFlag from 'hooks/useFeatureFlag';
+
+// utils
+import {
+  getPermissionActionMessage,
+  getPermissionActionSectionSubtitle,
+} from './utils';
+import { IPCAction } from 'typings';
 
 const ActionPermissionWrapper = styled.div`
   margin-bottom: 30px;
+  margin-left: 20px;
 
   &.last {
     margin-bottom: 0;
   }
 `;
 
+const StyledTitle = styled(Title)`
+  display: flex;
+  align-items: center;
+
+  ::after {
+    content: '';
+    flex: 1;
+    margin-left: 1rem;
+    height: 1px;
+    background-color: ${colors.grey400};
+  }
```

Comment:
This is used to make the horizontal line following a title. Maybe I should add a comment to make that clear?

Reply:
Yeah good idea